packages/flutter/lib/src/widgets/shortcuts.dart

```diff
@@ -211,6 +211,16 @@ abstract class ShortcutActivator {
   ///    modifier key is pressed when the side variation is not important.
   bool accepts(RawKeyEvent event, RawKeyboard state);
 
+  /// Returns true if the event and keyboard state would cause this
+  /// [ShortcutActivator] to be activated.
+  ///
+  /// If the keyboard `state` isn't supplied, then it defaults to using
+  /// [RawKeyboard.instance].
+  static bool isActivatedBy(ShortcutActivator activator, RawKeyEvent event, [RawKeyboard? state]) {
```

Comment:
Why do we have an optional `state`? I don't see anywhere omitting it. Unless we're using the omitted version, let's enforce it and write "typically you should use" in the doc.

Reply:
It's not used in this PR, but I'm planning on using the omitted version in the menu code. If it weren't for needing to supply it for tests, I would just eliminate the optional parameter.